styles/main.css

```diff
@@ -28,7 +38,6 @@ header>nav>ul>li>a {
 }
 
 a {
-    list-style: none;

```

Comment:
list-style has no effect on `a` elements.  You need to apply this property to a list.